common/src/main/kotlin/com/sourceclear/agile/piplanning/objects/Things.kt

```diff
@@ -54,7 +54,7 @@ data class TicketO(
     val crossBoardDependents: Boolean
 )
 
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME)
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
```

Comment:
This is required so that the json property is not "@type", allowing frontend to model the notification object correctly.

Reply:
I'm not attached to the name, but changing this would require updating the tests + the new WebSockets code in here and the FE.

Reply:
I fixed the `NotificationsTest` as per recent commit. This would not change anything in the websocket changes since it is currently using `type` as property name right now. https://github.com/srcclr/sapling-frontend/pull/12/files#diff-6c58ae6ccad72d41e6c1b57e29ac7621L17

Reply:
Oh, I meant the other places we use `@type`. Don't worry, I'll fix those.